app/build.gradle

```diff
@@ -97,6 +97,7 @@ dependencies {
         exclude module: 'recyclerview-v7'
     }
     annotationProcessor 'androidx.room:room-compiler:2.2.6'
+    implementation 'de.hdodenhof:circleimageview:3.1.0'
```

Comment:
What is this new implementation for ?

Reply:
it simply adds a tag in the xml that allows to have a circular image. I use it to enhance the design of our layout.